layouts/joomla/form/field/color/slider.php

```diff
@@ -49,67 +57,89 @@
 $onclick      = $onclick ? ' onclick="' . $onclick . '"' : '';
 $preview      = $preview ? ' data-preview="' . $preview . '"' : '';
 $readonly     = $readonly ? ' readonly' : '';
+$saveFormat   = $saveFormat ? ' data-format="' . $saveFormat . '"' : '';
 $size         = $size ? ' size="' . $size . '"' : '';
 $validate     = $validate ? ' data-validate="' . $validate . '"' : '';
 
 $displayValues = explode(',', $display);
 $allSliders    = $display === 'full' || empty($display);
-$displayData   = ' data-display="' . ($display ? $display : 'full') . '"';
 
 HTMLHelper::_('stylesheet', 'system/fields/joomla-field-color-slider.min.css', ['version' => 'auto', 'relative' => true]);
 HTMLHelper::_('script', 'system/fields/joomla-field-color-slider.min.js', ['version' => 'auto', 'relative' => true]);
+
+Text::script('JFIELD_COLOR_ERROR_CONVERT_HSL');
+Text::script('JFIELD_COLOR_ERROR_CONVERT_HUE');
+Text::script('JFIELD_COLOR_ERROR_NO_COLOUR');
```

Comment:
Much as I hate to see the word spelled as color lets not confuse thing by using both ;)
```suggestion
Text::script('JFIELD_COLOR_ERROR_NO_COLOR');
```